### DISCUSSION TITLE
is there something like autocomplete/suggest?

### DISCUSSION THREAD

**Discussion by Kreijstal** [mikaelmello/inquire#172] [Q&A]
something like https://github.com/prompt-toolkit/python-prompt-toolkit
![](https://raw.githubusercontent.com/prompt-toolkit/python-prompt-toolkit/master/docs/images/ptpython.png)
Where you type and you get suggestions and you can tab through them but after autocompleting it leaves you unbothered

**Comment by mikaelmello** [comment]
That's pretty interesting. I had also never heard of a visual suggestions tab in the terminal. Would you happen to know which tech they use to implement that?
